### ISSUE TITLE
Set up repo to allow pull requests

### ISSUE THREAD

**Issue by TeaDrinkingProgrammer** [common-fate/docs#20]
Hi there.

I would like to change some things and make a pull request, but I run into a lot of issues:
- The README is the default readme from Astro
- The repo seems to use PNPM, but when I run PNPM install with the latest version, the lock file is completely overhauled and updated from version 6 to 9, suggestion out-of-date packages.
- When I run `pnpm run dev`, I get this error on the `localhost` page: `Could not process image metadata for .../docs/src/assets/cf-new-access-screenshot.png.`. When I look in the asset folder myself, my image viewer also gives the same error. I am running Fedora 41 with KDE (Linux).

I understand these are a lot of questions, so feel free to ask me to elaborate on some points. The docs are well-structured, and with a bit of work and maybe a "contribute" button, the barrier-to-entry for contributing can be lowered significantly

**Comment by chrnorm** [comment]
Hi @TeaDrinkingProgrammer, sorry to hear that you ran into these issues when trying to get set up to contribute.

I've just fixed the README and lockfile issues in #21. Thanks for your feedback here.

> When I run pnpm run dev, I get this error on the localhost page: Could not process image metadata for .../docs/src/assets/cf-new-access-screenshot.png.. When I look in the asset folder myself, my image viewer also gives the same error. I am running Fedora 41 with KDE (Linux).

I suspect this is because you do not have [Git LFS](https://git-lfs.com) installed. We use Git LFS to store images so that the repo doesn't get bloated over time. Could you please follow the [Git LFS getting started guide](https://git-lfs.com) and let me know if it fixes the issue for you?

For reference, my dev machine is properly configured with LFS and I get the following output when I run `git lfs status` from the root of the repo in my terminal:

```
❯ git lfs status
On branch main
Objects to be pushed to origin/main:


Objects to be committed:


Objects not staged for commit:
```

You should see something similar to this. Alternatively, if the issue persists, could you please let me know which localhost path you see it at?

**Comment by TeaDrinkingProgrammer** [comment]
@chrnorm Thank you for your swift response! There is still a tiny issue though. When I reset to the main branch, I get this message with one file changed:
```shell
git reset --hard origin/main
HEAD is now at bd09923 Fix Git LFS image reference (#22)
Encountered 1 file that should have been a pointer, but wasn't:
        src/content/docs/granted/images/browser-extension-automatic-code-confirm.png
```

All the other stuff now works. Thank you so much!
